pino.js

```diff
@@ -354,6 +359,9 @@ function asErrValue (err) {
 
 function genLog (z) {
   return function LOG (a, b, c, d, e, f, g, h, i, j, k) {
+    if (this.filter && !this.filter(a, b, c, d, e, f, g, h, i, j, k)) {
```

Comment:
When I read `if (this.filter(...))` I expect `true` to mean "yes, the filter applied." Returning `false` for the skip seems backward to me.